Launcher/Classes/Controllers/TagsController.swift

```diff
@@ -8,7 +8,7 @@ class TagsController {
         if let launchPath = Constants.FilePaths.Bash.tags {
             let outputStream = CommandsCore.CommandTextOutputStream()
             outputStream.textHandler = {text in
-                tags.append(contentsOf: text.components(separatedBy: "\n"))
+                tags.append(contentsOf: text.components(separatedBy: "\n").filter { !$0.isEmpty })
```

Comment:
@q231950 Not sure why it is needed. Sometimes we get a lot of empty strings after actual result.

Reply:
I guess somewhere in the script an empty line is generated.